Source/IGListCollectionContext.h

```diff
@@ -92,6 +92,25 @@ NS_ASSUME_NONNULL_BEGIN
                                                       atIndex:(NSInteger)index;
 
 /**
+ Dequeues a cell from the UICollectionView reuse pool.
+ 
+ @param nibName           The name of the nib file.
+ @param bundle            The bundle in which to search for the nib file.
+ @param cellClass         The class of the cell you want to dequeue.
+ @param sectionController The section controller requesting this information.
+ @param index             The index of the cell.
+ 
+ @return A cell dequeued from the reuse pool or newly created.
+ 
+ @note This method uses a string representation of the cell class as the identifier.
+ */
+- (__kindof UICollectionViewCell *)dequeueReusableCellFromNibName:(NSString *)nibName
+                                                           bundle:(nullable NSBundle *)bundle
+                                                        cellClass:(Class)cellClass
```

Comment:
I started implementing this and I realized that the initializer for `IGListSingleSectionController` now has 3 `nullable` parameters since we no longer need the `cellClass` for nib cells. Is this too confusing for the user?

```
- (instancetype)initWithCellClass:(nullable Class)cellClass
                          nibName:(nullable NSString *)nibName
                           bundle:(nullable NSBundle *)bundle
                   configureBlock:(IGListSingleSectionCellConfigureBlock)configureBlock
                        sizeBlock:(IGListSingleSectionCellSizeBlock)sizeBlock NS_DESIGNATED_INITIALIZER;
```